### PR TITLE
feat: Add visualization step for predictions in AutoML Forecasting model training sample

### PR DESCRIPTION
**Pull request by thehardikv**
Added a step for visualizing the predictions generated by a trained model in the AutoML Forecasting model training sample.

### CODE REVIEW

**Comment by ivanmkc** [comment]
Awesome! I learned something new.

File: ai-platform-unified/notebooks/unofficial/sdk/Vertex_AI_SDK_AutoML_Forecasting_Model_Training_Example.ipynb [hunk 1]
```diff
@@ -435,6 +435,84 @@
       "execution_count": null,
```
Comment (ivanmkc) [inline ai-platform-unified/notebooks/unofficial/sdk/Vertex_AI_SDK_AutoML_Forecasting_Model_Training_Example.ipynb:435]: <pre class="ql-syntax" spellcheck="false">Line #49.          "templateId": "067f70d2-8cd6-4a4c-a099-292acd1053e8",
</pre><p>How do you ensure that this template won't be deleted?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/399/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

**Review by ivanmkc — APPROVED** [review]
One comment but LGTM

File: ai-platform-unified/notebooks/unofficial/sdk/Vertex_AI_SDK_AutoML_Forecasting_Model_Training_Example.ipynb (hunk 1 above)
Comment (thehardikv) [reply ai-platform-unified/notebooks/unofficial/sdk/Vertex_AI_SDK_AutoML_Forecasting_Model_Training_Example.ipynb:435]: The solution is not 100% satisfying: We created the template, we own the template, hence we'll make sure not to delete it.

**Pull request merged**